src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -1,11 +1,10 @@
 package cromwell.engine.backend.jes
 
-import java.io.File
 import java.nio.file.Paths
 
 import cromwell.binding._
 import cromwell.binding.values.WdlFile
-import cromwell.engine.backend.jes.JesBackend.{JesInput, JesOutput}
+import cromwell.engine.backend.jes.JesBackend.{JesOutput, localFilePathFromRelativePath}
```

Comment:
I usually don't see imports of functions in our codebase... is that considered good practice in Scala?